wordpress-mobile/WordPress-Android
# Issue/fix gb paragraph scroll issue

Fixes [this](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2056) scrolling issue in Block Editor.

Looks like `AppBarLayout`, when placed in `RelativeLayout` was preventing web-based content of editor from scrolling. Recreating the layout using `ConstraintLayout` instead of `RelativeLayout` fixed the issue. Thanks to @marecar3 for finding the solution.

To test:
- Open a long post in the block editor.
- Select an existing paragraph or add a new one, try scrolling the editor.
- Confirm that editor scrolls as expected.

In Aztec:
- Open a post, tap on + icon to open the media picker.
- Notice that it works as expected, and the content of the editor is dimmed and not actionable when media picker is open.

PR submission checklist:
